Playbooks/AS-PagerDuty-Integration/README.md

```diff
@@ -0,0 +1,129 @@
+
+![PagerDuty_Create_Integration_Key_2](Images/PagerDuty_Create_Integration_Key_2.png)              
+                                                                                       
+                                                                                         
+Select an escalation policy for your service. In this example, the default is used.
+
+![PagerDuty_Create_Integration_Key_3](Images/PagerDuty_Create_Integration_Key_3.png)
+
+
+Select a noise reduction option for your service. The "**Intelligent**" option is reccomended.
+
+![PagerDuty_Create_Integration_Key_3.5](Images/PagerDuty_Create_Integration_Key_3.5.png)
+
+                                                                                           
+Finally, select the Microsoft Azure option under the Integrations section, then click "**Create Service**".
+
+![PagerDuty_Create_Integration_Key_4](Images/PagerDuty_Create_Integration_Key_4.png)
+
+                                                                                                  
+You will be redirected to the service you have created. Under the Integrations tab view, you will find your integration key. This will be needed for the deployment of this playbook.
+                                                                                                  
+![PagerDuty_Create_Integration_Key_5](Images/PagerDuty_Create_Integration_Key_5.png)
+                                                                                           
+                                                                                                 
+#
+### Deployment                                                                                                         
+                                                                                                        
+To configure and deploy this playbook:
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fazuredeploy.json)
+
+[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fazuredeploy.json)                                             
+
+Click the “**Deploy to Azure**” button at the bottom and it will bring you to the custom deployment template.
+
+In the **Project Details** section:
+
+* Select the “**Subscription**” and “**Resource Group**” from the dropdown boxes you would like the playbook deployed to.  
+
+In the **Instance Details** section:   
+
+* **Playbook Name**: This can be left as “AS-PagerDuty-Integration” or you may change it.  
+
+* **Integration Key**: Enter the value of the PagerDuty integration key created from the first section.
```

Comment:
A small change, can we clearly indicate here that the key provided in this section would appear in plain text. To ensure the security of this key, best practice is to save the same in key-vault and utilize the same from there. This step is not currently taken care by this deployment.